Contribution Guidelines.md

```
@@ -2,14 +2,31 @@
 
 Thank you for your interest in contributing to **BabyNest**! We appreciate your efforts in making this project better. Please follow these best practices to ensure smooth collaboration.
 
+
+## Prerequisites: Local Setup
+
+Before contributing, you need to set up your development environment:
+
+1.⁠ ⁠*Frontend Setup*: Follow the instructions in [Setup.md](./Setup.md) for React Native setup (Node.js, npm/pnpm, Android/iOS tools).
+2.⁠ ⁠*Backend Setup*: Install Python 3.8+, Flask, and dependencies as outlined in [Backend README](./Backend/README.md).
+3.⁠ ⁠*Verify Installation*: Run the frontend and backend servers locally to ensure everything is working:
+   - *Frontend*: ⁠ npm start ⁠ or ⁠ npx react-native run-android/run-ios ⁠
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)